electron/electron.js

```diff
@@ -7,8 +7,9 @@ const isDev = process.env.NODE_ENV == "dev"
 function createWindow() {
   // Create the browser window.
   const mainWindow = new BrowserWindow({
-    width: 800,
```

Comment:
For now, it's ok but manage these via env file later.

Reply:
Sure, I'll fix this in Electron-specific Ticket.

Reply:
from memory this can be controlled in package.json

Reply:
Okay